qemu/tests/pci_bridge_hotplug_unplug.py

```diff
@@ -0,0 +1,157 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_misc
+
+
+def disk_hotplug(test, params, vm, session, image, drive_format, parent_bus):
+    """
+    Hotplug new disk.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param vm: VM object
+    :param session: VM session
+    :param image: Name of the new disk
+    :param drive_format: Drive subsystem type (virtio, scsi, usb2)
+    :param parent_bus: Bus(es), in which this device is plugged in
+    """
+    def check_usb_in_guest():
+        """
+        Check USB in guest
+        """
+        output = session.cmd(params["chk_usb_cmd"],
+                             float(params["cmd_timeout"]))
```

Comment:
`cmd_timeout` be used not only once, it is better to assign to a param, so no need to get from `params` everytime.

Reply:
Modified, Thanks!